wiki/BBCode/zh.md

```diff
@@ -1,390 +1,356 @@
+`[centre]` 居中标签用于将文本与显示区域的中心对齐。这通常用于标题、标题或诗歌的文体效果。如果放置在 `[quote]` 标记内或周围，引号块内的文本将居中，但样式行等部件不会居中。
 
 ### URL
 
-添加一条链接。
+```
+[url=LINK]文本[/url]
+```
 
-如果你不想使用链接文本，则不需要使用此标记。osu!论坛 将会在帖子中自动把链接转化成 URL 。
+`[url]` 链接标签用于将普通文本转换成可点击的超链接。
 
-- 专用按钮： ![URL](img/url.png)
-- 注：
-  - `LINK` 参数不应该使用引号包围。
-- 语法：
-  - `LINK` 参数为链接地址。
-  - `TEXT` 参数为链接文本。
-  ```
-  [url=LINK]TEXT[/url]
-  ```
+*注意：如果不想自定义超链接的标题文本，则不需要使用这个标签。论坛编辑器会自动将合适的 URL 转换为链接。*
 
-### 玩家简介
+要创建带有 `[url]` 标记的超链接，则用户必须指定两个参数：要显示的链接文本和要导航到的网站的特定 URL。前者必须填写在开始和结束标签之间，后者必须写在 `LINK` 参数内，并且不带引号 (`"`)。如果未指定文本，则显示的超链接文本将默认为 URL 的名称。
 
-使用玩家名生成玩家简介链接。
+工具栏按钮：![URL 按钮](img/url.png "URL")
 
-**非常不推荐**使用这个标签！不推荐使用的原因是，如果某个玩家获得了一个 [osu!supporter 标签](/wiki/osu!supporter)，那么他的用户名可以被更改。一旦玩家更改了他的用户名，这个链接就会失效。
+### 主页
 
-建议使用 [URL](#url) 标签代替（使用该玩家的 ID）。
+```
+[profile=userid]玩家名[/profile]
+```
 
-- 专用按钮： N/A
-- 注：
-  - 不推荐使用！
-  - `USER` 参数必须被指定。
-- 语法：
-  - `USER` 为该用户的名字。
-  ```
-  [profile]USER[/profile]
-  ```
+`[profile]` 主页标签通过读取设定的玩家名或玩家 ID，提供一个可以访问玩家主页的链接。与 `[url]` 标签不同的是，`[profile]` 标签在光标悬停在链接上时，会显示玩家卡片。
 
-### Google
+*注意：玩家 ID 即玩家个人主页链接 `/users/` 之后的一串数字。*
 
-使用所提供的文本自动链接到 Google 搜索查询。
+如果只通过玩家 ID 来指定玩家时，需要在开始和结束标签之间添加占位文本。这么做的话，会在实际页面上显示玩家的名字，而不是占位文本。如果只通过玩家名来指定玩家时，只要玩家修改了名称，链接将失效。
 
-- 专用按钮： N/A
-- 注：
-  - 要知道，不同的人可能得到不同的搜索结果。
-    - 一些搜索结果会由于语言或位置的原因而被隐藏。
-- 语法：
-  ```
-  [google]...[/google]
-  ```
+### 列表
 
-### 手气不错
+```
+[list=TYPE]
+[*]选项 1
+[*]选项 2
+[*]选项 3
+[/list]
+```
 
-用所提供的文本，使用 Google 的 *手气不错* 按钮，**添加一个指向该页面的直链**。
+`[list]` 标签用于在 osu! 论坛中自动格式化两种不同类型的列表，通过使用方括号中的星号 (`[*]`) 来标示列表中的新项目（如上所示）。默认情况下，这将创建一个简单的项目符号列表。
 
-- 专用按钮： N/A
-- 注：
-  - 要知道，不同的人可能得到不同的搜索结果。
-    - 一些搜索结果会由于语言或位置的原因而被隐藏。
-- 语法：
-  ```
-  [lucky]...[/lucky]
-  ```
+如果 `TYPE` 参数被指定（无论是什么值），会创建有序列表。
 
-### 列表
+*注意：尽管这会导致格式问题，但是 BBCode 格式的列表依旧可以层层嵌套，也可以放在一起，*
+
+工具栏按钮：![无序列表按钮](img/list.png "无序列表") ![有序列表按钮](img/list-numbered.png "有序列表")
+
+### 邮箱
+
+```
+[email=ADDRESS]文本[/email]
+```
+
+`[email]` 邮箱标签创建一个可点击的超链接，点击它会在默认邮件程序中创建一封新的电子邮件，并会预先填充收件地址字段。
 
-将文本格式化为一个列表。
-
-在 BBCode 中，列表需要有两个部分：包装器和项目符号。制作列表需要把项目符号放在包装器内。
-
-- 专用按钮：
-  - 无序列表： ![无序列表](img/list.png)
-  - 有序列表： ![有序列表](img/list-numbered.png)
-  - 列表项目符号： N/A
-  - 列表类型： N/A
-- 注：
-  - 所有的项目符号必须被列表或列表类型的标签包围。
-  - 这些是一些合法的 `TYPE` 参数：
-    - (空白) - 无序列表
-    - `1` - 数字
-    - `a` - 字母（小写）
-    - `A` - 字母（大写）
-    - `i` - 罗马数字（小写）
-    - `I` - 罗马数字（大写）
-  - 默认情况下，项目符号列表使用的是无序列表。
-  - 嵌套的列表有已知漏洞。
-- 语法：
-  - `TYPE` 是上面列出的类型之一。
-  - 如果没有定义 `TYPE`，则将使用无序列表。
-  ```
-  [list=TYPE]
-  [*]...
-  [/list]
-  ```
+为了创建超链接，需要指定两个参数：在 `ADDRESS` 参数中指定有效的邮箱地址，`text` 参数中指定超链接的标题文本。如果不指定 `text` 参数，超链接则无法正常显示。
 
 ### 图片
 
-**从在线图片源显示一个图像。**
+```
+[img]图床链接[/img]
+```
+
+`[img]` 图片标签用于在 osu! 论坛中显示在线图片。若要使用这个标签，则用户必须粘贴图片的直链（在 `图床链接` 参数中指定）。本地文件路径（比如 `C:\Users\Name\Pictures\image.jpg`）**将不起作用**。
+
+*注意：网站的 URL **不等同于**图片的直链。*
+
+如果想获得图片的直链，则必须访问图片来源的网站，并将光标悬停在图像上，右键单击，选择“复制图片地址”，之后，将复制好的图片直链粘贴在开始和结束标签之间。
+
+为了获得图像地址，必须导航到其来源的网站，将鼠标悬停在图像上，右键单击图像，然后选择“复制图像地址”。然后，地址应该被复制并粘贴在标签之间。
```

Comment:
I assume one of these was only meant to be a draft or something?

Reply:
removing draft paragraph

Reply:
and ty for the screenshot